### DISCUSSION TITLE
Clang-Format includes git-clang-format, which is missing functionality "--staged"

### DISCUSSION THREAD

**Discussion by jhgorse** [Homebrew/discussions#3069] [Everyday usage]
Not sure where to report. As issue in hombrew-core?

The clang-format formula
https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/clang-format.rb
https://formulae.brew.sh/formula/clang-format

installs git-clang-format which lacks the "--staged" functionality. So it is stale.

How can I report it to the maintainer of that formula?

Cheers,
Joe

**Comment by carlocab** [comment]
Formulae do not have maintainers. Does `clang-format` support the `--staged` flag if you build it yourself from source outside of Homebrew?

**Reply by jhgorse** [reply]
Technically, it's `git-clang-format` with the `--staged` flag.
https://github.com/llvm/llvm-project/blob/llvmorg-14.0.0-rc3/clang/tools/clang-format/git-clang-format

It is not in 13.0.1 so that is probably why it's not in the bottled version. Trying --HEAD.
